### PR TITLE
[Fix] Fix bg point filter 

### PR DESCRIPTION
**Pull request by xiliu8006**
fix the issue(#347)

### CODE REVIEW

File: configs/3dssd/3dssd_kitti-3d-car.py [hunk 1]
```diff
@@ -87,8 +87,8 @@
 ]
 
 data = dict(
-    samples_per_gpu=4,
-    workers_per_gpu=4,
+    samples_per_gpu=2,
```
Comment (Wuziyi616) [inline configs/3dssd/3dssd_kitti-3d-car.py:90]: This config file seems strange to me. It doesn't have `batch_size x num_GPUs` in its name (such as `2x8`)?

**Review by Wuziyi616 — COMMENTED** [review]
The issue said this [line](https://github.com/open-mmlab/mmdetection3d/blob/7f9b186e82d7945cb3bfe6fb4bc110dcb1d570ae/mmdet3d/models/dense_heads/ssd_3d_head.py#L426) is also strange. Is it also caused by this bug?

File: configs/3dssd/3dssd_kitti-3d-car.py (hunk 1 above)
Comment (xiliu8006) [reply configs/3dssd/3dssd_kitti-3d-car.py:90]: I forget to modify it, sorry

Comment (Wuziyi616) [reply configs/3dssd/3dssd_kitti-3d-car.py]: What about the filename? Should we change it to something like `3dssd_**4x8**_kitti-3d-car.py`?

File: mmdet3d/datasets/pipelines/transforms_3d.py [hunk 2]
```diff
@@ -1127,11 +1127,11 @@ def __call__(self, input_dict):
         points = input_dict['points']
         gt_bboxes_3d = input_dict['gt_bboxes_3d']
 
-        gt_bboxes_3d_np = gt_bboxes_3d.tensor.numpy()
-        gt_bboxes_3d_np[:, :3] = gt_bboxes_3d.gravity_center.numpy()
+        gt_bboxes_3d_np = gt_bboxes_3d.tensor.clone().numpy()
```
Comment (ZwwWayne) [inline mmdet3d/datasets/pipelines/transforms_3d.py:1130]: Add comments indicating why clone is necessary here.

**Review by ZwwWayne — APPROVED** [review]

**Pull request merged**